src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -411,8 +449,17 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
           />
         )}
         {renderTableHeader.map((cellProps, index) => {
+          const _cellProps = isResizable ?
+            { 
+              ...cellProps, 
+              _onResize: (width: number) => this.handleCellResize(index, width),
+              _onResizeComplete: () => this.persistCellSizes(),
+              size: this.cellSizes[index],
+              isResizable 
+            } : cellProps;
```

Comment:
Same with my arguments above, the truthiness of `isResizable` should determine whether the two callback are used so might as well just pass them in always.